### PR TITLE
feat: Implement create a job

### LINKED ISSUE #20: feat: Implement the option to create a job
## Feature Request

**Is your feature request related to a problem? Please describe.**
Organizations with a vacancy should be able to post job openings.

### PR DESCRIPTION
**Pull request by vrushti-mody**
Closes #20

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+GOOGLE_CLIENT_ID=904529028821-e6no686045d8d3qboe2rftc6pjmvmc4s.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=e6hQukHkr8vfeOCx4FBhKg-p
+GOOGLE_CALLBACK_URL=http://localhost:3000/account/google/callback
```
Comment (vidhi-mody) [inline .env:3]: Remove this file

File: public/scripts/menu-form.js [hunk 2]
```diff
@@ -193,4 +193,32 @@ $(document).ready(function () {
     e.preventDefault()
     $(this).parent('div').remove() //Remove field html
   })
+
+  var addButton5 = $('.add_button5') //Add button selector
+  var wrapper5 = $('.field_wrapper5') //Input field wrapper
+
+  //Once add button is clicked
```
Comment (und3fined-v01d) [inline public/scripts/menu-form.js:200]: This should be in a separate file

File: routes/post/index.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const mime = require('mime-types')
 const router = require('express').Router()
 const { v4 } = require('uuid')
 const formParser = require('../../utils/parsers/form-parser')
-const { Post } = require('../../models')
+const { Post, Job } = require('../../models')
```
Comment (und3fined-v01d) [inline routes/post/index.js:8]: Please follow alphabetical order here

File: routes/post/index.js [hunk 4]
```diff
@@ -72,4 +72,31 @@ router.get('/delete/:id', async (req, res, next) => {
...
 
+router.get('/job', (req, res, next) => {
+  res.render('post/job', {
+    title: req.app.config.name,
+    user: req.session.user
+  })
+})
+
+router.post('/job', async (req, res, next) => {
+  const newJob = new Job({
```
Comment (und3fined-v01d) [inline routes/post/index.js:83]: this is not the correct implementation.

1) there should be a check for the user to be company
2) the job model has a company field...how do we know which company files this job posting else?
3) job _id must be pushed to companies jobs array too and that should be should be saved

File: views/components/sidebar.ejs [hunk 5]
```diff
@@ -41,6 +41,10 @@
           <i class="fa fa-plus"></i>&nbsp;
           <a title="Share your thoughts" href="/post/upload/">Post</a>
         </li>
+        <li class="list-group-item">
+            <i class="fa fa-briefcase"></i>&nbsp;
+            <a title="Create a job" href="/post/job/">Post a Job</a>
```
Comment (und3fined-v01d) [inline views/components/sidebar.ejs:46]: check indentation

File: views/post/job.ejs [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                    <form action="/post/job" method="post">
+                      <h3 style="color: black; text-align: left;">CREATE JOB OPENING</h3>
+                      <div class="form-group">
+                        <label for="role">Role*</label>
+                        <input type="name" class="form-control" id="role" placeholder="Enter the role for the job here" name="role" required/>
+                      </div>
+                      <div class="form-group">
+                        <label for="experience">Experience required*</label>
+                        <select class="form-control" name="experience" style="margin-top: 2%;" required>
+                            <option>0-1 years</option>
```
Comment (und3fined-v01d) [inline views/post/job.ejs:26]: This is not what we have done in file upload. Please that as template. The entire template is wrong

**Pull request merged**